engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -319,19 +304,16 @@ private void renderShadowMap() {
     public void renderWorldReflection() {
         PerformanceMonitor.startActivity("Render World (Reflection)");
 
-        DefaultRenderingProcess.getInstance().beginRenderReflectedScene();
-        GL11.glCullFace(GL11.GL_FRONT);
+        LwjglRenderingProcess.getInstance().beginRenderReflectedScene();
         playerCamera.setReflected(true);
 
-        playerCamera.lookThroughNormalized();
+        playerCamera.lookThroughNormalized(); // we don't want the reflected scene to be bobbing or moving with the player
         backdropRenderer.render(playerCamera);
         playerCamera.lookThrough();
 
         chunkShader.activateFeature(ShaderProgramFeature.FEATURE_USE_FORWARD_LIGHTING);
 
         if (renderingConfig.isReflectiveWater()) {
-            GL11.glEnable(GL_LIGHT0);
```

Comment:
Hmmm... I just noticed through issue #656 that according to begla forward lighting is used to light transparent objects, something I do not normally see as I'm doing tests on a core-only engine setup. 

I must therefore rethink this aspect of this PR and probably submit a new commit that re-enable this light. 

Also, it turns out GL_LIGHT0 is the exception to the rule and by default is set to white. So, I probably can't see its effect just because I haven't tested it in the appropriate situation.